dataikuapi/dss/apideployer.py

```diff
@@ -238,6 +249,34 @@ def save(self):
                 body = self.settings)
 
 
+class DSSAPIDeployerInfraStatus(object):
+    """
+    The status of an API Deployer infrastructure.
+
+    Do not create this directly, use :meth:`~dataikuapi.dss.apideployer.DSSAPIDeployerInfra.get_status`
+    """
+    def __init__(self, client, infra_id, light_status):
+        self.client = client
+        self.infra_id = infra_id
+        self.light_status = light_status
+
+    def list_deployments(self):
+        """
+        Returns the deployments that are deployed on this infrastructure
+
+        :returns: a list of deployments
+        :rtype: list of :class:`dataikuapi.dss.apideployer.DSSAPIDeployerDeployment`
+        """
+        return [DSSAPIDeployerDeployment(self.client, deployment.id) for deployment in self.light_status["deployments"]]
```

Comment:
I think you need to get the id with `deployment["id"]`?

Reply:
Oh wow, cannot believe I let that slip 🤦